CMakeLists.txt

```diff
@@ -5,8 +5,9 @@ PROJECT(PlusBuild)
 # Configure the build to work (although with limited functionalities) if only
 # src directory of the repository is available
 
-SET(CMAKE_CXX_STANDARD 11 CACHE INTERNAL "Use C++11 standard")
-MARK_AS_ADVANCED(CMAKE_CXX_STANDARD)
+IF(NOT DEFINED CMAKE_CXX_STANDARD)
+  SET(CMAKE_CXX_STANDARD 11 CACHE STRING "C++ standard version")

```

Comment:
I know we didn't before, but should this check also set `CMAKE_CXX_STANDARD_REQUIRED` ?

Reply:
You could do what is done here: https://github.com/Slicer/ChestImagingPlatform/commit/bc173fdba7db8ebac9119bcef10dffaac20f7f03

Reply:
Sure, I've updated the PR to use something similar to what @lassoan suggested to manage the CMAKE_CXX_STANDARD value